distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -91,6 +110,7 @@ public void testCswRegistryUpdate() throws Exception {
                         .replaceAll("urn:uuid:2014ca7f59ac46f495e32b4a67a51276",
                                 "urn:uuid:2014ca7f59ac46f495e32b4a67a51280")
                         .replace("Node Name", "New Node Name")
```

Comment:
Is it dangerous to hard-code dates like this?  Will it cause inconsistent test results later on?